docs/tails_guide.rst

```diff
@@ -41,14 +41,17 @@ create a Tails USB stick. Here are some links to help you out:
 -  `Install onto a USB stick or SD
    card <https://tails.boum.org/doc/first_steps/installation/index.en.html>`__
 -  `Create & configure the persistent
-   volume <https://tails.boum.org/doc/first_steps/persistence/configure/index.en.html>`__
+   volume 
+<https://tails.boum.org/doc/first_steps/persistence/configure/index.en.html>`__
```

Comment:
e.g. this line wrapping is all messed up. Note that you only need one `_` to indicate a link (aka [reference name](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#reference-names)) in ReST.

In general, I prefer [external hyperlink targets](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#hyperlink-targets) to internal hyperlink targets because:

1. They are reusable
2. They make the text easier to read (no URLs all over the place) and also more closely resemble the rendered output
3. They help avoid line length problems like the one you have here.